tinygrad/runtime/graph/hcq.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, jit_cache: List[ExecItem], input_rawbuffers: List[Buffer], va
 
     for j,ji in enumerate(jit_cache):
       enqueue_dev = ji.prg.dev if (is_exec_prg:=isinstance(ji.prg, CompiledRunner)) else Device[ji.bufs[1].device] #type:ignore
+      assert isinstance(enqueue_dev, HCQCompiled) and issubclass(enqueue_dev.hw_compute_queue_t, HWQueue) and enqueue_dev.hw_copy_queue_t is not None
```

Comment:
Err, how sure are you that this is right? What if it doesn't have a copy queue?

Reply:
Sorry I missed that QCOM had a graph